demos/http/http_demo_s3_download/demo_config.h

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (C) 2020 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+#ifndef DEMO_CONFIG_H
+#define DEMO_CONFIG_H
+
+/**************************************************/
+/******* DO NOT CHANGE the following order ********/
+/**************************************************/
+
+/* Logging related header files are required to be included in the following order:
+ * 1. Include the header file "logging_levels.h".
+ * 2. Define LIBRARY_LOG_NAME and  LIBRARY_LOG_LEVEL.
+ * 3. Include the header file "logging_stack.h".
+ */
+
+/* Include header that defines log levels. */
+#include "logging_levels.h"
+
+/* Logging configuration for the Demo. */
+#define LIBRARY_LOG_NAME     "DEMO"
+#define LIBRARY_LOG_LEVEL    LOG_INFO
+
+#include "logging_stack.h"
+
+/************ End of logging configuration ****************/
+
+/**
+ * @brief Host for S3 GET Object access.
+ */
+#define S3_PRESIGNED_HOST               "Please update the host for the demo"
+
+/**
+ * @brief The length of the host for S3 GET Object access.
+ */
+#define S3_PRESIGNED_HOST_LENGTH        ( sizeof( S3_PRESIGNED_HOST ) - 1 )
+
+/**
+ * @brief Presigned URL path for S3 GET Object access.
+ */
+#define S3_PRESIGNED_GET_PATH           "Please update the path for the demo"
+
+/**
+ * @brief The length of the presigned URL path for S3 GET Object access.
+ */
+#define S3_PRESIGNED_GET_PATH_LENGTH    ( sizeof( S3_PRESIGNED_GET_PATH ) - 1 )
+
+/**
+ * @brief Port of [resigned URL for S3 GET Object access.
+ *
+ * In general, port 443 is for HTTPS connections.
+ */
+#define S3_PRESIGNED_PORT               443
+
+/**
+ * @brief Root CA certificate associated with the S3 server certificate for
+ * TLS server authentication.
+ *
+ * This is the Baltimore Cybertrust root CA certificate.
+ */
+#define SERVER_CERTIFICATE                                               \
```

Comment:
S3 might soon be changing to use an Amazon issued root CA. So we should do what the other demos for root CAs here.